components/Dashboard/CardGrid.js

```diff
@@ -143,25 +279,59 @@ const WiPCard = styled.div`
   border: 1px solid rgba(255, 255, 255, 0.3);
   position: relative;
   margin: 2.5%;
+  display: flex;
+  .header {
+    height: 30%;
+    width: 100%;
+    background: rgba(255, 255, 255, 0.3);
+    display: flex;
+    justify-content: flex-start;
+    border-radius: 15px 15px 0 0;
+    padding: 8px;
+  }
 `;
+
 const ContainerHeadingTwo = styled.h2`
-  font-size: 1rem;
-  color: #fff;
-  margin-left: 15px;
+  font-size: 0.8rem;
+  color: #000;
+  margin: 0;
+  padding: 5px;
+  display: flex;
+  align-items: center;
+  height: 15%;
+
+  .iconstyle {
+    margin-right: 5px;
+    color: #000;
+  }
 `;
 const ParamCircle = styled.div`
-  width: 50px;
-  height: 50px;
+  width: 40px;
+  height: 40px;
   border-radius: 30px;
   background-color: orange;
+  opacity: 80%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  bottom: 12%;
+  right: 8%;
+`;
+const DoneParamCircle = styled.div`
+  width: 40px;
+  height: 40px;
+  border-radius: 30px;
+  background-color: green;
+  opacity: 80%;
```

Comment:
`ParamCircle` and `DoneParamCircle` share a lot of identical styles. To prevent code duplication, you could parameterize `ParamCircle` with a prop (e.g. `status="done"`):

```js
const ParamCircle = styled.div`
  // ...
  background-color: ${({status})=>status==="done"?"green":"orange"};
  // ...
`;
```
You could do the same exercise with your various card components (DRY = don't repeat yourself!).